tests/test_smoke.py

```diff
@@ -832,6 +832,26 @@ def test_public_bucket(self, tmp_public_storage_obj, store_type):
         out = subprocess.check_output(['sky', 'storage', 'ls'])
         assert tmp_public_storage_obj.name not in out.decode('utf-8')
 
+    @pytest.mark.parametrize(
+        'nonexist_bucket',
+        [f's3://{str(uuid.uuid4())}', f'gs://{str(uuid.uuid4())}'])
```

Comment:
There is a small chance the bucket exists. In the test, have a while loop generate bucket names until it finds one that exists?

Reply:
Fixed!